### ISSUE TITLE
[Feature]: verified flathub app and snap app

### ISSUE THREAD

**Issue by dodollama** [simplex-chat/simplex-chat#4382]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Linux

### Feature

i know simplex is there in flathub but this important when i come to IM i request to have a verified badge in flathub so the trust can be built.

**Comment by pm4rcin** [comment]
It's verified now, just check Flathub ;)

**Comment by dodollama** [comment]
yes it is not using wayland and does not looks good macos version have rounded corners but linux variant does not offer that
